tests/saliency_metrics/metrics/insertion_deletion/test_insertion_deletion.py

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+import pytest
+import torch
+import torch.nn as nn
+from torchvision.transforms import GaussianBlur
+
+from saliency_metrics.metrics.insertion_deletion.insertion_deletion import InsertionDeletion
+from saliency_metrics.models import CUSTOM_CLASSIFIERS
+
+
+@CUSTOM_CLASSIFIERS.register_module()
+class TestNet(nn.Module):
+    def __init__(self) -> None:
+        super().__init__()
+        self.features = nn.Sequential(
+            nn.Conv2d(in_channels=3, out_channels=5, kernel_size=3, padding=1),
+            nn.MaxPool2d(2, 2),
+            nn.ReLU(inplace=True),
+        )
+        self.classifier = nn.Sequential(nn.Linear(16 * 16 * 5, 10))
+        self.initialize_weights()
+
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
+        x = self.features(x)
+        x = x.view(-1, 16 * 16 * 5)
+        x = self.classifier(x)
+        return x
+
+    def initialize_weights(self) -> None:
+        for i, x in enumerate(self.parameters()):
```

Comment:
See my last comment. PLEASE AVOID RANDOM INITIALIZATION. EVEN IF RANOM INITIALIZATION IS NEEDED, PLEASE DO NOT INITIALIZE THE NETWORK LIKE THIS. SEE `torch.nn.init`. :)